docs/pages/documentation/3-querying-with-graql/dml.md

```diff
@@ -0,0 +1,320 @@
+knowledge base and return an [answer](./query.html#answer) with variables bound to concepts mentioned in the
+[variable patterns](./query.html#variable-pattern).
+
+If a [match](#match) is provided, then the [insert query](#insert-query) will operate for every
+[answer](./query.html#answer) of the [match](#match) and return one [answer](./query.html#answer) for each
+[match](#match) [answer](./query.html#answer).
+
+Within the [variable patterns](./query.html#variable-pattern), the following [properties](./query.html#property) are
+supported:
+
+## isa
+
+`$x isa $A` creates a new direct instance of the _type_ `$A` and binds it to `$x`.
+
+## relationship
+
+`$r ($A1: $x1, ..., $An, $xn)` will, for each `$Ai: $xi`, add a new role-player `$xi` to the _relationship_ `$r`
+directly playing the _role_ `$Ai`.
+
+## has (data)
+
+`$x has <identifier> $y via $r` will relate the _attribute_ `$y` of _attribute type_ `<identifier>` to `$x`, binding the
+_relationship_ to `$r`.
+
+If `<identifier>` is a key for the direct type of `$x`, then the resulting relationship will be a key relationship.
+
+The `via $r` is optional. Additionally, a literal [value](./query.html#value) can be used instead of a
+[variable](./query.html#variable):
+
+```graql-test-ignore
+$x has <identifier> <value>;
+```
+
+which is equivalent to:
+
+```graql-test-ignore
+$x has <identifier> $_;
+$_ val <value>;
+```
+
+<!-- TODO: Describe without referring to relationships? -->
+> `has` is syntactic sugar for a particular kind of relationship.
+>
+> If `<identifier>` is not a key for the direct type of `$x`, it is equivalent to:
+>
+> ```graql-test-ignore
+> $r (has-<identifier>-owner: $x, has-<identifier>-value: $y) isa has-<identifier>;
+> $y isa <identifier>;
+> ```
+>
+> Otherwise, it is equivalent to:
+>
+> ```graql-test-ignore
+> $r (key-<identifier>-owner: $x, key-<identifier>-value: $y) isa key-<identifier>;
+> $y isa <identifier>;
+> ```
+
+## val
+
+`$x val <value>;` specifies that the _attribute_ `$x` should have [value](./query.html#value) `<value>`.
+
+## id
+
+`$x id <identifier>` will assign `$x` to an existing _concept_ with the given ID. It is an error if no such concept
+exists.
+
+## label
+
+`$A label <identifier>` will assign `$A` to an existing _schema concept_ with the given label. It is an error if no such
+schema concept exists.
+
+# Delete Query
+
+[`<match>`](#match) `delete` [`(<variable>, ...)`](./query.html#variable) `;`
+
+For every [answer](./query.html#answer) from the [match](#match), the [delete query](#delete-query) will delete the
+concept bound to every [variable](./query.html#variable) listed. If no [variables](./query.html#variable) are provided,
+then every variable mentioned in the [match](#match) is deleted.
+
+# Aggregate Query
+
+[`<match>`](#match) `aggregate` [`<aggregate>`](#aggregate) `;`
+
+An [aggregate query](#aggregate-query) applies the given [aggregate](#aggregate) to the [answers](./query.html#answer)
+of the [match](#match).
```

Comment:
of the `get`?

Reply:
This one is still `match`... for now anyway :P